README.rst

```diff
@@ -1039,7 +1039,7 @@ Run celery  worker (separate terminal window is needed):
 .. code-block:: shell
 
     # (cd tests)
-    celery -A openwisp2 worker -l info
+    celery -app openwisp2 worker
```

Comment:
I get it @pandafy . In that case I also don't feel the need for the change, because the rest of the command complies with the docs. The confusion got even clearer when earlier today I was going through ansible-openwisp's codebase and the commands seem perfectly fine with the docs. I'll revert that change. Thanks! :)

Reply:
so the cli command is no different than celery 4.x?

Reply:
Yes, @nemesisdesign . As correctly pointed out by @pandafy , our cli commands are indeed compatible with the docs and we won't be requiring the change of cli command. It's only the case that our command uses the -A option and a few additional tags which seems perfectly fine.

The tests worked fine after installing celery 5.0.5, in that case I think that's the only change we require in this patch to upgrade to celery 5. A similar approach can be taken in monitoring and radius I suppose, along with a check for dependency errors.